app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_secure_password
+  has_many :tutors
+  has_many :reservations, dependent: :destroy
+
+  validates :name, presence: true, length: { maximum: 50 }
```

Comment:
- Good job adding more validations and not only `presence: true` now, I invite you to do the same with all validations on other models too, this could help to avoid special characters/to few passwords, etc. I invite you to check the others and consider this and others' good practices on the validations. 🐴

Reply:
Hello @vichuge thank you for the suggestion, its implemented